src/app/home/banner/banner.component.scss

```diff
@@ -0,0 +1,19 @@
+.parallax-container {
+  @media (max-width: 600px) {
```

Comment:
can you change this to use materialize variables please??

https://github.com/gp4tech/website/blob/develop/src/app/layout/footer/footer.component.scss#L10